sources/packages/backend/libs/integrations/src/services/sfas/sfas-individual.service.ts

```diff
@@ -68,6 +68,10 @@ export class SFASIndividualService
     individual.birthDate = getISODateOnlyString(sfasIndividual.birthDate);
     individual.sin = sfasIndividual.sin;
     individual.pdStatus = sfasIndividual.pdStatus;
+    individual.ppdStatus = sfasIndividual.ppdStatus;
```

Comment:
To be fair, we need an adjustment during the student creation to consider ppd status from SFAS and that was not included in the ticket. 

We need to create a seperate ticket for the same. 

![image](https://github.com/bcgov/SIMS/assets/54600590/dee824b2-cc40-4e3a-8989-70aab670e535)

Reply:
I did that adjust in this ticket.